config/default.yaml

```diff
@@ -23,6 +23,8 @@ services:
     url: ''
   documentManagement:
     url: ''
+  cdam:
+    url: ''
```

Comment:
In order for this to work from local pointing to aat, you might need to update the endpoint secret in adoption aat keyvault and use it [here](https://github.com/hmcts/adoption-web/blob/6b7447621ac131d863770b2e197a1292555579ee/src/main/modules/properties-volume/index.ts#L57)

Reply:
I added cdam url to Azure hat endpoints. made changes in index.ts to use that url. It is working fine